test/test.js

```diff
@@ -0,0 +1,232 @@
+mocha.ui('bdd');
+mocha.reporter('html');
+
+describe('hermes', function () {
+    'use strict';
+
+    var assert = chai.assert;
+
+    describe('setup', function () {
+        it('should attach a list to the provided element', function () {
```

Comment:
e.g.: it looks like all your tests start with "should" - you can just remove that and not lose any value e.g.: 'should attach a list' -> 'attaches a list'